django_cloud_deploy/cli/prompt.py

```diff
@@ -714,6 +714,13 @@ def prompt(self, console: io.IO, step: str,
             creds = self.auth_client.create_default_credentials()
         else:
             creds = self.auth_client.get_default_credentials()
+            if not creds:
+                console.ask(
+                    ('Warning: You are using a service account to authenticate '
+                     'gcloud, or your existing credentials file is either '
```

Comment:
either xxx? I believe you missed a word.

Reply:
My bad. Fixed.